meta-directive.js

```diff
@@ -0,0 +1,35 @@
+/*!
+ * Copyright (c) 2016 Digital Bazaar, Inc. All rights reserved.
+ */
+define([], function() {
+
+'use strict';
+
+function register(module) {
+  module.directive('brMeta', factory);
+}
+
+/* @ngInject */
+function factory() {
+  return {
+    replace: true,
+    restrict: 'E',
+    templateUrl: requirejs.toUrl('bedrock-angular-meta/meta-directive.html'),
+    controller: Ctrl,
+    controllerAs: '$ctrl'
+  };
+
+  /* @ngInject */
+  function Ctrl($rootScope, brMetaService, $location) {
+    var self = this;
+    self.meta = brMetaService.meta[$location.url()];
```

Comment:
I think this needs to change dynamically. So we probably need to expose `brMetaService.meta` and `$rootScope.route` or something like that ... and do the dereferencing in the template.